crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -190,9 +211,9 @@ pub(crate) struct TypeInference<'db> {
 }
 
 impl<'db> TypeInference<'db> {
-    pub(crate) fn expression_ty(&self, expression: ScopedExpressionId) -> Type<'db> {
-        self.expressions[&expression]
-    }
+    // pub(crate) fn expression_ty(&self, expression: ScopedExpressionId) -> Type<'db> {
+    //     self.expressions[&expression]
+    // }
```

Comment:
here I went with leaving only `try_expression_ty`, other places I `Option`'d the existing methods. Not sure what pattern makes sense (if any)